presto-native-execution/presto_cpp/presto_protocol/presto_protocol-json-hpp.mustache

```diff
@@ -32,7 +32,7 @@
 #include <folly/Format.h>
 #include <utility>
 #include "velox/common/encode/Base64.h"
-#include "presto_cpp/external/json/json.hpp"
+#include "presto_cpp/external/json/nlohmann/json.hpp"
```

Comment:
Why is that?
On my mac setup I don't have "nlohmann" folder in "presto/presto-native-execution/presto_cpp/external/json".
I have only the header:
```
spershin@spershin-mbp ~ % ls /Users/spershin/git/presto/presto-native-execution/presto_cpp/external/json
json.hpp
```

Reply:
@spershin https://github.com/prestodb/presto/pull/20619